packages/wonder-blocks-icon-button/components/icon-button.js

```diff
@@ -116,6 +116,11 @@ export type SharedProps = {|
  * light (whether the IconButton will be rendered on a dark background),
  * disabled , test ID, and custom styling.
  *
+ * Note: the size of an IconButton matches the size of the icon it wraps 
+ * which is 24x24 pixels.  The focus ring, which is displayed on hover and 
+ * focus, is much larger but does not affect its size.  This matches the 
+ * behavior of Button.
```

Comment:
Sort of - the focus ring on button is much less pronounced - it's easier to put two buttons next to each other, but you can't put two iconbuttons next to each other (without spacing).

Reply:
A person shouldn't be putting buttons next to each other either.  If they did, the focus ring of one button would (and does) overlap the other, obviously not to the extent that would occur with IconButtons that adjacent without spacing.

Reply:
True! I think we should provide clarity in the documentation around this as I'm not sure the best practices are apparent from what we've presented (especially in the context of a toolbar).